CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 0.3.0 (February 6, 2020)
+
+* General API improvements
+* Adherence to protocol format described in
```

Comment:
Do we want to succintly mention a few of the changes involved in the protocol changes, e.g. user ID changes, serialization changes / RFC 8017 compliance ?